libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/room/RoomMember.kt

```diff
@@ -16,16 +16,22 @@
 
 package io.element.android.libraries.matrix.api.room
 
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
```

Comment:
Why do we need it to be Parcelable?

Reply:
It's used in a NavTarget, and I think those need to be Parcelable.